android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -70,17 +73,22 @@
                 else
                     Snackbar.make(v, context.getResources().getString(R.string.no_comments), Snackbar.LENGTH_SHORT).show();
             });
+
+            feedImageView.setOnClickListener(v -> {
+                zoomImage(Utils.parseImageUri(feedItems.get(getPosition()).getFullPicture()));
+            });
         }
     }
 
-    public FeedAdapter(Context context, AdapterCallback adapterCallback, List<FeedItem> feedItems) {
+    public FeedAdapter(Context context, AdapterCallback adapterCallback, ZoomableImageModule zoomableImageModule, List<FeedItem> feedItems) {
         super(feedItems);
         this.feedItems = feedItems;
         this.context = context;
         try {
             this.mAdapterCallback = adapterCallback;
+            this.zoomableImageModule = zoomableImageModule;
         } catch (ClassCastException e) {
-            throw new ClassCastException("Activity must implement AdapterCallback.");
+            throw new ClassCastException("Activity must implement Callbacks.");
```

Comment:
Not good, throwing runtime exceptions based on class casting exception is like using Java like JavaScript. Java is type safe, there should be no need to convert an object in raw type and then check if it is of the correct type. I see that you did not write this implementation, but don't add more to it. There shouldn't be a `ClassCastException` if we expect the right type in the arguments.

Please also open an issue to fix the existing code and removing the exception check